consensus/polybft/system_state.go

```diff
@@ -13,9 +13,9 @@ import (
 // ValidatorInfo is data transfer object which holds validator information,
 // provided by smart contract
 type ValidatorInfo struct {
-	Address             types.Address `json:"address"`
 	Stake               *big.Int      `json:"stake"`
 	WithdrawableRewards *big.Int      `json:"withdrawableRewards"`
+	Address             types.Address `json:"address"`
```

Comment:
I've quickly checked, we should be fine with this change (in terms there is no fear that it is a breaking change).